### DISCUSSION TITLE
Solução CORS refresh-token

### DISCUSSION THREAD

**Discussion by uberlei** [balta-io/7187#52] [Perguntas e Respostas] [answered]
Pessoal caso alguém tenha problema com refresh-token como eu tive basta tirar o null do body da chamada http;
Da erro:
refreshToken() { return this.http.post(`${this.url}/accounts/refresh-token`,null, { headers: this.composeHeaders() } ); }
Não da erro:
refreshToken() {return this.http.post(`${this.url}/accounts/refresh-token`, { headers: this.composeHeaders() })}

**Comment by andrebaltieri** [comment, marked as answer]
Muito obrigado @uberlei <3
